gradle-baseline-java/src/main/groovy/com/palantir/baseline/plugins/BaselineIdea.groovy

```diff
@@ -80,7 +80,19 @@ class BaselineIdea extends AbstractBaselinePlugin {
             }
         }
 
-        project.getTasks().named("idea").configure(idea -> idea.doLast(cleanup))
+        project.getTasks().named("idea").configure(idea -> {
+            idea.doFirst(_t -> project.getLogger().warn("""
+                DEPRECATED: Using `./gradlew idea` is no longer recommended, some functionality may not function as expected. 
```

Comment:
We still have a lot of occurrences in our docs that use `./gradlew idea` (https://github.com/search?q=org%3Apalantir+%22.%2Fgradlew+idea%22&type=code). Should we update these before merging?

Reply:
We probably should, but that is a serious amount of manual work.